sentry-gateway.go

```diff
@@ -67,30 +68,40 @@ func run(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-
-	tmplPath, err := cmd.Flags().GetString("template")
-	if err != nil {
-		return err
+	if dsn != "" {
+		raven.SetDSN(dsn)
```

Comment:
I believe that we should return error when `dsn` is empty.

Reply:
The point was to let `raven` use `SENTRY_DSN` var if `SetDSN` was not called explicitly. However I now see that it doesn't return an error when `SENTRY_DSN` is empty, just logs a warning. So I'll do the check here.

Reply:
That's make sense. Thank you!